### ISSUE TITLE
DK Summon Gargoyle Target selection

### ISSUE THREAD

**Issue by tripleslash** [dalaranwow/dalaran-wow#4871]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When summoning your gargoyle with the summon gargoyle ability, the gargoyle doesn't attack the target unless you move your character (and therefore your pets) around.

**Expected behaviour**: When using Summon Gargoyle, a gargoyle should descend from the sky and start bombarding the target that has the Gargoyle DoT on it immediately. After the cast is over, the gargoyle should ascend into the sky and disappear.

**Steps to reproduce the problem**:

1. Summon Gargoyle
2. Dont move but continue to attack the boss
3. Gargoyle does nothing and visual is bugged (doesn't ascend/descend from the sky and doesn't pick target).

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you for the report @tripleslash 
This issue should be fixed by #2213
Will reopen this report if it isn't.

**Comment by tripleslash** [comment]
Target selection and gargoyle fly in still not working

**Comment by swiftdeath** [comment]
Can confirm that it is still broken. It will only attack when it takes some sort of damage or debuff .

**Comment by thanhtotti** [comment]
Confirm that it still doesn't work properly

**Comment by Lothloryen** [comment]
Hello there,

Gargoyle will receive several improvements within the next scheduled restart, fixing also this issue.

Thank you.
